src/View/View.php

```diff
@@ -1,16 +1,16 @@
 <?php
 /**
- * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
- * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
```

Comment:
Undo these http to https changes. This header needs to stay consistent across all files.

You can make a separate PR to change it for all files if you want.